packages/generators/src/addon-generator.ts

```diff
@@ -37,6 +33,17 @@ const addonGenerator = (
     templateFn: (instance: any) => Record<string, unknown>,
 ): Generator.GeneratorConstructor => {
     return class extends Generator {
+        public logger: Record<string, (string) => void>;
+        public getPackageManager: (val: void) => string;
```

Comment:
let's move it under `utils`

Reply:
so we can add more utils in future

Reply:
yep lets do it

Reply:
need to union types for all utils, it is nightmare 😞

Reply:
You can use `any` right now, after migrate on ts or jsdocs we will fix it

Reply:
thanks